app/assets/stylesheets/application.scss

```diff
@@ -1,17 +1,17 @@
+$govuk-use-legacy-palette: true;
+$govuk-typography-use-rem: false;
+
 @import "govuk_publishing_components/all_components";
 
 .app-email-option {
-  @include core-24;
-  margin-top: $govuk-gutter-half;
-  margin-bottom: $govuk-gutter-half;
+  @include govuk-font(24);
+  margin-top: govuk-spacing(3);
+  margin-bottom: govuk-spacing(3);
   line-height: 1.5;
```

Comment:
I think line height can be specified as part of the `govuk-font` mixin, e.g: `@include govuk-font(24, $line-height: 1.5)`

Reply:
Thanks, fixed.